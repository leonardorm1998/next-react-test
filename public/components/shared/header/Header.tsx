import React from 'react';
import { SHeader } from './style';

export default () => {

  return (
    <SHeader>
      <h1>Criando estrutura do projeto</h1>
    </SHeader>
  );
};
