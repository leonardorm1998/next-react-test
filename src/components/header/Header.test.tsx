import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import React from 'react'
import renderer from 'react-test-renderer'

import Header from './'

Enzyme.configure({ adapter: new Adapter() })

const Component = <Header />
const wrapper = shallow(Component)

describe('Should render Header component', () => {
  it('#Should have title', () => {
    const title = wrapper.find('h1')

    expect(title.length).toBe(1)
    expect(title.text()).toEqual('Header Example')
  })

  it('should render correctly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
