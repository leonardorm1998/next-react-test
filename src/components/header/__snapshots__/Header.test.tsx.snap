// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component should render correctly 1`] = `
<header
  className="sc-AykKC iLTvVK"
>
  <h1>
    Header Example
  </h1>
  <img
    src="test-file-stub"
  />
</header>
`;
